frontend/src/component/feature/FeatureStrategy/FeatureStrategyEdit/change-request-conflict-data.ts

```diff
@@ -14,18 +12,7 @@ type ChangeRequestConflictCreatedData = {
 };
 
 export const getChangeRequestConflictCreatedData = (
-    changeRequests:
-        | {
-              state: ChangeRequestType['state'];
-              id: ChangeRequestType['id'];
-              features: {
-                  name: IChangeRequestFeature['name'];
-                  changes: (Pick<IFeatureChange, 'action'> & {
-                      payload: { id?: number | string };
-                  })[];
-              }[];
-          }[]
-        | undefined,
+    changeRequests: ChangeRequestType[] | undefined,
```

Comment:
@thomasheartman can you please double check that this is sane? From what I've seen the only reason for this semi-type was to satisfy the types in tests. I think we should try to adapt our tests to our code and not the other way around, so this seems fine to me, but still want to double check.

Reply:
You're right! The only reason for it was to make it easier to test (so you don't have to generate a full change request for each test). 

I'm happy for you to revert it as long as the tests still pass 💁🏼  

I ... think I agree with adapting tests to code, but in this case, it also made be more clear about **what** the function needs instead of just saying "change request". So I'm undecided. Anyway, yeah, feel free to change it 😄

Reply:
I think I understand what you mean, since strictly speaking it may only need a subset of a change request 🤔 maybe this could be tackled in the future by using sub/base types that are extended to the full type only when needed.

However in general I'm leaning towards just specifying the full type every time, since that's easier, more maintainable, and it could just be the case that there are no current real use cases other than tests, like in this case.

Thank you for your input 🙌